source/stylesheets/cart.sass

```diff
@@ -155,7 +159,7 @@ a.continue-shopping
   border-color: transparent
   color: $primary-text-color
   display: block
-  font-size: 12px
+  font-size: $luna-font-base-px
```

Comment:
This line can probably be removed since it shouldn't be changing from desktop -> mobile

Reply:
Not sure what you mean-- this is the only font-size rule for the `a.continue-shopping` element. Can you point me to where you see the sizes for different devices?

Reply:
Ah github was being deceptive, I thought this was being defined in a different way. Nevermind :)

Reply:
deceptive hidden lines !